### ISSUE TITLE
RIP USTVGO

### ISSUE THREAD

**Issue by smellyboot90** [interlark/ustvgo_downloader#62]
its gone now

**Comment by interlark** [comment]
Yeah, looks like it's gone :) There's [123tv service](https://github.com/interlark/123tv-iptv), it works not that good as USTVGO and has less channels, but it works.
<hr/>

![image](https://user-images.githubusercontent.com/20641837/213893928-b54d9697-6794-46bb-9a20-6a2c1db761de.png)

**Comment by Nostradamus1973** [comment]
Good morning Everyone,
I doubt their coming back, word on the street is they were getting 20 million visitors per month which got them noticed because of the advertisement revenue. Supposedly they will RE-open under a different name and resume their activities.
I'm more of a realist, I'm thinking that they are down(possibly forever) and we should start looking for viable replacements and then see if Andy has the time and is up for taking on another project.

**Comment by Nostradamus1973** [comment]
Andy,

Is there a certain criteria that a site needs innately in order to use your script?
I did a cursory search and found a few places that look interesting:

[TV Zinos](https://www.tvzinos.com/) Only has a few mid to lower to way-lower level entertainment.

[Ustream](https://www.ustream.to/2): This site looks amazing which makes me think that it wouldn't/couldn't work because I'm not sure how it could go un-noticed.

[OkLiveTV](http://oklivetv.com/north-america/united-states/): Pretty cool selection of mid-to-lower level entertainment, there's almost too much to sift through.

I'm sure these have been looked at; if they haven't, would any of them work?

**Comment by simvande** [comment]
Can you try tv247.us?

**Comment by r34p3r112** [comment]
Is it possible to adapt to https://tv.ustvgo.live/?

**Comment by Nostradamus1973** [comment]
@r34p3r112 That site looks nice. Is that the new re-iteration of ustvgo? Or just a coincidence that the channel set is very similar to ustvgo's?

**Comment by r34p3r112** [comment]
Not sure about where it came from, but full transparency, I was checking on news about the original site and I think I mis-typed...

**Comment by dahartigan** [comment]
> Is it possible to adapt to https://tv.ustvgo.live/?

Looks like a fake, the streams refer to ustvgo.tv which is now defunct, and some channels are using twitch as the source (Fox News for example)

**Comment by 3v3rm4rt3lL** [comment]
What about this site, https://teleclub.xyz/  , they ask for "active" the service each 24 hours to be validated your ip when requests a channel. Is possible  to automatice something here?

**Comment by interlark** [comment]
@3v3rm4rt3lL, looks like it's not a TV playlist, it's just a bunch of randomly scraped playlist that updates from time to time, so not suitable for IPTV.

**Comment by 3v3rm4rt3lL** [comment]
Oh, got it. Good to know. Thanks

**Comment by s66jones** [comment]
Maybe daddyhd or 123tv?

**Comment by interlark** [comment]
@s66jones, [123tv.live](http://123tv.live) already implemented - [123tv-iptv](https://github.com/interlark/123tv-iptv).
[daddylivehd.sx](https://daddylivehd.sx) looks promising.

**Comment by interlark** [comment]
[123tv](http://123tv.live/) also RIP.
As far as I know the best IPTV provider today is [daddylivehd.sx](https://daddylivehd.sx/).
Maybe it's worth to port this project for this provider.

**Comment by dahartigan** [comment]
I am in favor of a port to any provider that works :)

**Comment by Nostradamus1973** [comment]
Good morning Everyone,
I'm looking at a github project that encompasses all available free sites, it's called Cabernet
Looks pretty bad ass, but, I won't know until I get it running in Docker lol, but, it looks really cool, provides a master m3u and epg
Thanks and have a good day.

**Comment by jakizak** [comment]
> Andy,
> 
> Is there a certain criteria that a site needs innately in order to use your script? I did a cursory search and found a few places that look interesting:
> 
> [TV Zinos](https://www.tvzinos.com/) Only has a few mid to lower to way-lower level entertainment.
> 
> [Ustream](https://www.ustream.to/2): This site looks amazing which makes me think that it wouldn't/couldn't work because I'm not sure how it could go un-noticed.
> 
> [OkLiveTV](http://oklivetv.com/north-america/united-states/): Pretty cool selection of mid-to-lower level entertainment, there's almost too much to sift through.
> 
> I'm sure these have been looked at; if they haven't, would any of them work?

If you go to https://go.basevik.com/api/tvsource/streams/ you can pull an entire list of the streams used... surely helpful to someone... I'm going to try and build a custom M3u playlist from these and see where I get to.

**Comment by jakizak** [comment]
> > Andy,
> > Is there a certain criteria that a site needs innately in order to use your script? I did a cursory search and found a few places that look interesting:
> > [TV Zinos](https://www.tvzinos.com/) Only has a few mid to lower to way-lower level entertainment.
> > [Ustream](https://www.ustream.to/2): This site looks amazing which makes me think that it wouldn't/couldn't work because I'm not sure how it could go un-noticed.
> > [OkLiveTV](http://oklivetv.com/north-america/united-states/): Pretty cool selection of mid-to-lower level entertainment, there's almost too much to sift through.
> > I'm sure these have been looked at; if they haven't, would any of them work?
> 
> If you go to https://go.basevik.com/api/tvsource/streams/ you can pull an entire list of the streams used... surely helpful to someone... I'm going to try and build a custom M3u playlist from these and see where I get to.

Sorry, meant to add, that is from TV Zinos.
